android/RNKeys.gradle

```diff
@@ -83,7 +83,15 @@ def loadKeys() {
       def nodeModulesDir = getNodeModulesDir()
       exportCommand = exportCommand + nodeModulesDir + "/node_modules/react-native-keys/keysAndroid.js"
     }
-    def proc = ["/bin/sh", "-c", exportCommand].execute()
+
```

Comment:
I think, we also need to replace `export` with `set` as cmd dont know about `export`

Reply:
Yes, that might be correct but this works currently.

Reply:
if you don't replace `export` with `set` then example of this project wont work, can you please add conditional export and set, so i can merge this PR?

Reply:
Alright I will

Reply:
@moses-20 ?